### DISCUSSION TITLE
update vscode

### DISCUSSION THREAD

**Discussion by costadamorte** [topgrade-rs/topgrade#981] [Q&A] [answered]
Hi everybody
When I try to update Visual Studio Code with topgrade in debian, the system write an error, becouse the aplication can't openned with root permissions. Is there a topgrade configuration in order to update or avoid update vscode?
thanks

**Comment by SteveLauC** [comment]
Hi, how did you get VS Code installed, system package manager?

**Comment by lucaspar** [comment]
You'll see that error if you run topgrade itself as root, which you shouldn't do.

To elaborate on it, there are many package managers that shouldn't run as root, and they will inherit the user that started topgrade. If you must run as root, consider specifying only the steps to run, not the ones to skip:

```
topgrade --only [step]
```

**Comment by costadamorte** [comment, marked as answer]
Thanks very much
I decided to disable vscode
`[misc]
disable = ["vscode"]
`
